app/views/home/about.html.erb

```diff
@@ -97,6 +97,7 @@
 
   <h2 id="downvotes">Downvote Explanations</h2>
   <p>
+  Users may downvote stories and comments after their account has 50 karma.
```

Comment:
This should probably use the `User.MIN_KARMA_TO_DOWNVOTE` constant.

Reply:
I've done the suggestion you suggested